cypress.config.ts

```diff
@@ -8,6 +8,14 @@ if (!process.env.RUN_AGAINST_LOCAL_SERVICES) {
   dotenv.config({ path: import.meta.dirname + '/integration-tests/.env' });
 }
 
+if (process.env.RUN_AGAINST_LOCAL_SERVICES === '1') {
+  process.env.SUPERTOKENS_API_KEY = process.env.SUPERTOKENS_API_KEY ?? 'bubatzbieber6942096420';
```

Comment:
can we read these from `packages/services/server/.env.template` instead of having it hardcoded?  same for the url